eframe/src/epi.rs

```diff
@@ -211,6 +211,11 @@ pub struct NativeOptions {
     /// `egui` doesn't need the stencil buffer, so the default value is 0.
     pub stencil_buffer: u8,
 
+    /// Use hardware acceleration if available. On macOS, this will possibly
+    /// use a dedicated GPU which will lead to higher power consumption.
+    /// The default value is `Some(true)`
+    pub hardware_acceleration: Option<bool>,
```

Comment:
Do you know what `None` means? It is not documented at https://docs.rs/glutin/latest/glutin/struct.ContextBuilder.html#method.with_hardware_acceleration

Reply:
on mac, glutin only sets the relevant attribute (mentioned below) if it is some, otherwise it just skips setting that attribute.

from https://developer.apple.com/documentation/appkit/1436213-opengl_pixel_format_attributes/nsopenglpfaaccelerated 

A Boolean attribute. If present, this attribute indicates that only hardware-accelerated renderers are considered. If not present, accelerated renderers are still preferred.

so, hw acceleration preferred by default (None). if `Some(true)`, only hw acceleration contexts are considered. but ofcourse, someone with mac needs to test it out just to be sure.

Reply:
Interesting - so the current default (before and after this PR) means that we ONLY consider hardware-accelerated renderers, which could explain https://github.com/emilk/egui/issues/1358#issuecomment-1091893038

Perhaps the default should be `None` to allow software rendered backends too.